Logger.lua

```diff
@@ -55,24 +51,19 @@ end
 Logger.Path       = ""
 Logger.Namespace  = "Logger"
 Logger.Console    = false
-Logger.DefaultLvl = 1
+Logger.DefaultLvl = 3
 
 Logger.Type = {
-  { Name = "TRACE", Color = "32" },
-  { Name = "DEBUG", Color = "36" },
-  { Name = "INFO.", Color = "34" },
-  { Name = "WARN.", Color = "33" },
-  { Name = "ERROR", Color = "31" },
-  { Name = "FATAL", Color = "35" },
-  { Name = "OTHER", Color = "30" }
+  { Name = "OTHER", Color = "30" }, { Name = "TRACE", Color = "32" },
+  { Name = "DEBUG", Color = "36" }, { Name = "INFO.", Color = "34" },
+  { Name = "WARN.", Color = "33" }, { Name = "ERROR", Color = "31" },
+  { Name = "FATAL", Color = "35" }
 }
 
 local function StrToLogLevel(str)
   if type(str) == "string" then
     for i, v in ipairs(Logger.Type) do 
-      if (str:upper() == v.Name) or (str:upper() == v.Name:sub(1, #v.Name-1)) then 
-        return i 
-      end
+      if (str:upper() == v.Name:gsub("%.", "")) then return i end
```

Comment:
i changed the old method of checking a string towards v.name and v.name -1 character with checking a string towards v.name with the "."s removed

cambie el metodo antiguo de chequear un string contra v.name y v.name -1 caracter con chequear el mismo string con v.name pero con los "."s quitados